CheckDrive.Web/CheckDrive.Web/Views/Cars/Create.cshtml

```diff
@@ -2,105 +2,84 @@
-            </div>
-
-            <div class="form-group row">
-                <div class="col-md-6 mb-3">
-                    <label asp-for="MeduimFuelConsumption" class="control-label">100 km uchun yoqilg'i me'yori [litr]</label>
-                    <input asp-for="MeduimFuelConsumption" class="form-control" id="meduimFuelConsumption" />
-                    <span asp-validation-for="MeduimFuelConsumption" class="text-danger"></span>
-                </div>
-                <div class="col-md-6 mb-3">
-                    <label asp-for="Mileage" class="control-label">Bosib o'tilgan masofasi [km]</label>
-                    <input asp-for="Mileage" class="form-control" />
-                    <span asp-validation-for="Mileage" class="text-danger"></span>
-                </div>
-            </div>
-
-            <div class="form-group row">
-                <div class="col-md-6 mb-3">
-                    <label asp-for="OneYearMediumDistance" class="control-label">1 yillik bosib o'tish me'yori [km]</label>
-                    <input asp-for="OneYearMediumDistance" class="form-control" id="oneYearMediumDistance" oninput="calculateMonthlyDistance()" />
-                    <span asp-validation-for="OneYearMediumDistance" class="text-danger"></span>
-                </div>
-                <div class="col-md-6 mb-3">
-                    <label for="monthlyDistance" class="control-label">1 oylik bosib o'tish me'yori [km]</label>
-                    <input type="text" id="monthlyDistance" class="form-control" readonly />
-                </div>
-            </div>
-
-            <div class="form-group row">
-                <div class="col-md-6 mb-3">
-                    <label for="yearlyMediumFuel" class="control-label">1 yillik yoqilg'i me'yori [litr]</label>
-                    <input type="text" id="yearlyMediumFuel" class="form-control" readonly />
-                </div>
-                <div class="col-md-6 mb-3">
-                    <label for="monthlyMediumFuel" class="control-label">1 oylik yoqilg'i me'yori [litr]</label>
-                    <input type="text" id="monthlyMediumFuel" class="form-control" readonly />
-                </div>
-            </div>
-
-            <div class="form-group mt-3">
-                <a asp-action="Index" class="btn btn-outline-info">
-                    <i class="fa-solid fa-arrow-left-long"></i>
-                    Orqaga
-                </a>
-                <button class="btn btn-outline-success">
-                    <i class="fa fa-plus"></i> Yaratish
-                </button>
-            </div>
-        </form>
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+<link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.bundle.min.js"></script>
+
```

Comment:
This scipts be in Layout.cshtml or wwwRoot js files.

Reply:
Of course it is, but I put it because jquery didn't work